Search/Dpa.swift

```diff
@@ -11,34 +11,34 @@ import Foundation
 public class Dpa: NSObject {
 
     // MARK: Declaration for string constants to be used to decode and also serialize.
-    internal let kDpaLanguageKey: String = "LANGUAGE"
-    internal let kDpaLastUpdateDateKey: String = "LAST_UPDATE_DATE"
-    internal let kDpaRpcKey: String = "RPC"
-    internal let kDpaBuildingNumberKey: String = "BUILDING_NUMBER"
-    internal let kDpaPostcodeKey: String = "POSTCODE"
-    internal let kDpaUprnKey: String = "UPRN"
-    internal let kDpaMatchDescriptionKey: String = "MATCH_DESCRIPTION"
-    internal let kDpaEntryDateKey: String = "ENTRY_DATE"
-    internal let kDpaPostalAddressCodeKey: String = "POSTAL_ADDRESS_CODE"
-    internal let kDpaLocalCustodianCodeKey: String = "LOCAL_CUSTODIAN_CODE"
-    internal let kDpaStatusKey: String = "STATUS"
-    internal let kDpaBlpuStateCodeKey: String = "BLPU_STATE_CODE"
-    internal let kDpaOrganisationNameKey: String = "ORGANISATION_NAME"
-    internal let kDpaPostalAddressCodeDescriptionKey: String = "POSTAL_ADDRESS_CODE_DESCRIPTION"
-    internal let kDpaClassificationCodeDescriptionKey: String = "CLASSIFICATION_CODE_DESCRIPTION"
-    internal let kDpaXCoordinateKey: String = "X_COORDINATE"
-    internal let kDpaMatchKey: String = "MATCH"
-    internal let kDpaClassificationCodeKey: String = "CLASSIFICATION_CODE"
-    internal let kDpaTopographyLayerToidKey: String = "TOPOGRAPHY_LAYER_TOID"
-    internal let kDpaLocalCustodianCodeDescriptionKey: String = "LOCAL_CUSTODIAN_CODE_DESCRIPTION"
-    internal let kDpaBlpuStateCodeDescriptionKey: String = "BLPU_STATE_CODE_DESCRIPTION"
-    internal let kDpaDependentLocalityKey: String = "DEPENDENT_LOCALITY"
-    internal let kDpaLogicalStatusCodeKey: String = "LOGICAL_STATUS_CODE"
-    internal let kDpaYCoordinateKey: String = "Y_COORDINATE"
-    internal let kDpaThoroughfareNameKey: String = "THOROUGHFARE_NAME"
-    internal let kDpaAddressKey: String = "ADDRESS"
-    internal let kDpaPostTownKey: String = "POST_TOWN"
-    internal let kDpaBlpuStateDateKey: String = "BLPU_STATE_DATE"
+    internal static let LanguageKey: String = "LANGUAGE"
```

Comment:
Yeah, seems reasonable. I wonder if they could actually all be private. And possibly, moved to an extension so they can be moved out the way of the main useful code.